common/setup_precheck.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+parent: Common
+---
+
+# Setup Precheck

```

Comment:
Should we add a "Note" or "Warning"  section about possibly needing to do a setropts refresh?

Reply:
agree, added:

```
&nbsp;

{: .note}
> _Changes made using this script may require a SETROPTS REFRESH of the `FACILITY` class._
```

Reply:
makes sense; change made.

Reply:
Remove the word `of`. That was my mistake.

Reply:
I also should have caught that. change made.